### PR TITLE
Added SRC and INCLUDE

### PR DESCRIPTION
**Pull request by neerukulhari**
Ned to add cleanup file and update main.c

### CODE REVIEW

File: LJMD_serial/include/prototypes.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+#include <ctype.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+
+#ifdef __cplusplus
+extern "C"
```
Comment (iaraota) [inline LJMD_serial/include/prototypes.h]: @neerukulhari  why did you put extern "C"? We don't have C++ files

File: LJMD_serial/include/prototypes.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+#endif
+
+//IO functions
+extern int get_a_line(FILE *fp, char *buf);
```
Comment (iaraota) [inline LJMD_serial/include/prototypes.h]: I think also these extern are unnecessary

File: LJMD_serial/include/utilities.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+extern double wallclock();
+extern void azzero(double *d, const int n);
+extern double pbc(double x, const double boxby2);
```
Comment (iaraota) [inline LJMD_serial/include/utilities.h]: pbc is defined as "static double" in utilities.c

Comment (neerukulhari) [reply LJMD_serial/include/utilities.h]: I removed static everywhere

File: LJMD_serial/include/prototypes.h (hunk 1 above)
Comment (neerukulhari) [reply LJMD_serial/include/prototypes.h]: I thought we did in examples, so maybe we need in future

**Review by iaraota — APPROVED** [review]

**Pull request closed without merging**